components/Input.jsx

```diff
@@ -1,146 +1,128 @@
-import React, { PropTypes } from 'react'
+import React, { PropTypes, Component } from 'react'
 import classNamesBind from 'classnames/bind'
-import classNames from 'classnames'
 import styles from '@klarna/ui-css-components/src/components/input.scss'
+import * as programmaticFocus from '../lib/features/programmaticFocus'
+import * as fieldStates from '../lib/features/fieldStates'
+import * as inlinedIcon from '../lib/features/inlinedIcon'
```

Comment:
I think so. They are not really big. The problem is that having them as a decorator or something that you add doesn't work because they affect both markup and styles. This was the less messy solution.